airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/jsonl_parser.py

```diff
@@ -19,6 +19,9 @@ class JsonlParser(FileTypeParser):
     MAX_BYTES_PER_FILE_FOR_SCHEMA_INFERENCE = 1_000_000
     ENCODING = "utf8"
 
+    def check_config(self, config: FileBasedStreamConfig) -> Tuple[bool, Optional[str]]:
+        return True, None
```

Comment:
Do we expect a future revision to check additional items, or is there just nothing special to check for jsonl? 

Could we add a docstring here, one way or another?

Either

```suggestion
    def check_config(self, config: FileBasedStreamConfig) -> Tuple[bool, Optional[str]]:
        """No special config needed."""
        return True, None
```

or something like:

```suggestion
    def check_config(self, config: FileBasedStreamConfig) -> Tuple[bool, Optional[str]]:
        """TODO: In future, we may add additional config validation here."""
        return True, None
```

Thanks!